docs/development/contributing_guide.md

```diff
@@ -169,6 +169,21 @@ To increase the log level for the tests, set `SYNAPSE_TEST_LOG_LEVEL`:
 SYNAPSE_TEST_LOG_LEVEL=DEBUG trial tests
 ```
 
+By default, tests will use an in-memory SQLite database for test data. For additional
+help with debugging, one can use an on-disk SQLite database file instead, in order to
+review database state during and after running tests. This can be adjusted using the
+`SYNAPSE_TEST_PERSIST_SQLITE_DB` environment variable:
+
+```sh
+SYNAPSE_TEST_PERSIST_SQLITE_DB=test.db trial tests
+```
+
+The database file can then be inspected with:
+
+```sh
+sqlite3 test.db
```

Comment:
```suggestion
sqlite3 `_trial_temp` test.db
```
Probably not worth mentioning `_trial_temp-1`, `_trial_temp-2` etc here.

Reply:
I didn't take the suggestion exactly here, as it looks like it may have been a bit of a typo? Great point though!

Reply:
Ahh yes, confusing my markdown and shell. Thanks!